oss_src/process/process_win.cpp

```diff
@@ -39,32 +39,50 @@ bool process::launch(const std::string &cmd,
   STARTUPINFO startup_info;
   ZeroMemory(&proc_info, sizeof(PROCESS_INFORMATION));
   ZeroMemory(&startup_info, sizeof(STARTUPINFO));
+  startup_info.cb = sizeof(startup_info);
 
   char *c_arglist = convert_args(cmd, args);
 
+  logstream(LOG_INFO) << "Launching process using command: >>> " << c_arglist << " <<< " << std::endl;
+
+  // Set up the proper handlers.
+  startup_info.hStdError = GetStdHandle(STD_ERROR_HANDLE);
+  startup_info.dwFlags |= STARTF_USESTDHANDLES;
+
   BOOL ret;
+
+  ret = DuplicateHandle( GetCurrentProcess(), GetStdHandle(STD_ERROR_HANDLE),
+                         GetCurrentProcess(), &m_stderr_handle, 0, TRUE, DUPLICATE_SAME_ACCESS);
+
+  if(!ret) {
+    auto err = GetLastError();
+    logstream(LOG_WARNING) << "Failed to create duplicate stderr file handle: " << get_last_err_str(err) << std::endl;
+    logstream(LOG_WARNING) << "  Attempting to continue with : " << get_last_err_str(err) << std::endl;
+    m_stderr_handle = GetStdHandle(STD_ERROR_HANDLE);
+  }
+
   // For Windows, we include the cmd with the arguments so that a search path
   // is used for any executable without a full path
   ret = CreateProcess(NULL,
-      c_arglist,
-      NULL,
-      NULL,
-      FALSE,
-      CREATE_NO_WINDOW,
-      NULL,
-      NULL,
-      &startup_info,
-      &proc_info);
+      c_arglist, // command line
+      NULL,      // process security attributes
+      NULL,      // primary thread security attributes
+      TRUE,      // handles are inherited
+      0,         // creation flags
```

Comment:
I like that option.  Along with that, I think it should be passed as an argument from the server through to the spawning process.  This would eliminate the need for those very annoying environment variables.